src/main/kotlin/lotto/LottoNumber.kt

```diff
@@ -1,14 +1,17 @@
 package lotto
 
-data class LottoNumber(val number: Int) {
-    init {
-        require(number in LOTTO_NUMBER_RANGE) { LOTTO_NUMBER_OUT_OF_RANGE_MESSAGE }
-    }
-
+class LottoNumber private constructor(val number: Int) {
     companion object {
-        const val MAX = 45
-        const val MIN = 1
-        val LOTTO_NUMBER_RANGE: IntRange = (MIN..MAX)
-        const val LOTTO_NUMBER_OUT_OF_RANGE_MESSAGE = "$MIN 에서 $MAX 사이의 숫자만 가능합니다."
+        private const val MAX = 45
+        private const val MIN = 1
+        private val LOTTO_NUMBER_RANGE: IntRange = (MIN..MAX)
+        const val LOTTO_NUMBER_OUT_OF_RANGE_MESSAGE = "$MIN 에서 $MAX 사이의 숫자만 가능합니다. number:%d"
+        val LOTTO_NUMBER_POOL: List<LottoNumber> = LOTTO_NUMBER_RANGE.toList().map { LottoNumber(it) }
+
```

Comment:
IntRange는 list로 변환할 필요 없이 바로 map 연산을 활용할 수 있어요 :)
```suggestion
        val LOTTO_NUMBER_POOL: List<LottoNumber> = LOTTO_NUMBER_RANGE.map { LottoNumber(it) }

```